docs/cow-protocol/concepts/how-it-works/coincidence-of-wants.md

```diff
@@ -22,15 +24,15 @@ CoWs can be full or partial. In a partial CoW, only one of the trades in a batch
 A simple CoW is where two traders are each selling what the other is buying.
 They swap assets with each other instead of interacting with AMM liquidity pools.
 Simple CoWs can be either *complete*, where each trader is trading exactly the same amount that the other needs, or they can be *partial* where two traders are trading opposite assets but only one can have their full liquidity met through a CoW.
-In the latter case, the rest of the trade goes on-chain to finish filling. 
+In the latter case, the rest of the trade goes to an AMM to finish filling. 
```

Comment:
So wouldn't it be more correct to say "on-chain" (as we have it) than "to an AMM"? In your suggestion you're restricting it to just "AMM" rather than the more broad "on-chain"